include/zephyr/net/net_pkt.h

```diff
@@ -244,6 +244,8 @@ struct net_pkt {
 	uint8_t tx_timestamping : 1; /** Timestamp transmitted packet */
 	uint8_t rx_timestamping : 1; /** Timestamp received packet */
 #endif
+	uint8_t loopback: 1; /** Set to 1 if this packet is a loopback packet */
```

Comment:
Previous nit comment not addressed

```suggestion
	uint8_t loopback : 1; /** Set to 1 if this packet is a loopback packet */
```